examples/platform/silabs/MatterConfig.cpp

```diff
@@ -43,6 +43,9 @@
 
 #ifdef SIWX_917
 #include "wfx_rsi.h"
+#if SL_ICD_ENABLED
```

Comment:
`CHIP_CONFIG_ENABLE_ICD_SERVER` should work here.
We want to avoid using `SL_ICD_ENABLED` where it is not necessary.

Reply:
Okay got it